backend/apps/volontulo/forms.py

```diff
@@ -120,6 +127,51 @@ class Meta(object):
             'reserve_volunteers_limit',
         ]
 
+    def clean(self):
+        super(CreateOfferForm, self).clean()
+        self._clean_start_finish()
+        self._clean_recruitment_start_finish()
+        self._clean_reserve_recruitment_start_finish()
```

Comment:
instead of 3 almost the same methods create, please, one method, that will add errors for these 3 pairs of fields:
`def _clean_start_finish(start_field_slug, end_filed_slug, error_code)`